libraries/botbuilder-dialogs-adaptive/src/input/inputDialog.ts

```diff
@@ -111,9 +111,9 @@ export abstract class InputDialog extends Dialog {
 
     /**
      * Called when the dialog is started and pushed onto the dialog stack.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.